cloud_governance/aws/zombie_non_cluster/run_zombie_non_cluster_policies.py

```diff
@@ -306,3 +309,17 @@ def _organise_instance_data(self, resources: list):
                 volume['CreateTime'] = volume['CreateTime'].strftime("%Y-%m-%dT%H:%M:%S+00:00")
                 organize_data.append(volume)
         return organize_data
+
+    def get_ebs_cost(self, resource: any, resource_type: str, resource_hours: int):
+        ebs_cost = 0
+        if resource_type == 'ec2':
+            volume_ids = []
+            for block_device in resource:
+                volume_ids.append(block_device.get('Ebs').get('VolumeId'))
+            volumes = self._ec2_client.describe_volumes(VolumeIds=volume_ids)['Volumes']
+            for volume in volumes:
+                ebs_cost += self.resource_pricing.get_ebs_cost(volume_size=volume.get('Size'), volume_type=volume.get('VolumeType'), hours=resource_hours)
```

Comment:
Is it for extra EBS that attached for ec2 ?

Reply:
No. All attached ebs

Reply:
OK, so if I understand clear, you calculate just volumes or volumes that related to ec2, right ?

Reply:
Volumes cost. It is any volume(ebs, ec2_idle, ec2_stop)
Volume cost is same if we use or not